hooks/MockReturnTypeUpdater.php

```diff
@@ -51,7 +51,16 @@ public static function afterMethodCallAnalysis(
                 && $first_arg->right instanceof PhpParser\Node\Scalar\String_
                 && $first_arg->right->value[0] === '['
             ) {
-                $fq_class_name = $first_arg->left->class->getAttribute('resolvedName');
+                /** @var PhpParser\Node\Expr\ClassConstFetch $left */
```

Comment:
Psalm was complaining that `PhpParser\Node\Expr` doesn't have a `class` property here.  There's an `instanceof` check in the if condition above that makes sure that `$first_arg->left` is an instance of `PhpParser\Node\Expr\ClassConstFetch`, which does have the `$class` property: https://github.com/nikic/PHP-Parser/blob/master/lib/PhpParser/Node/Expr/ClassConstFetch.php#L12